T7Suite/AFRMap.cs

```diff
@@ -547,7 +546,7 @@ private byte[] LoadTargetAFRMapInBytes(float[] orginalmap, int size)
         private int[] fuelcorrectioncountermap;
         private float[] targetmap;
 
-        private bool _HasValidFuelmap = false;
+        //private bool _HasValidFuelmap = false;
```

Comment:
I thought that maybe it was here for a reason. Having as a comment makes easier to find it back. But if you prefer we can remove it. It will remain anyway in git.